x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_utils/get_viewable_detectors.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { each } from 'lodash';
```

Comment:
**nit**: I know you moved this function, but would be great to get rid of `each` from `lodash` 🙂

Reply:
Updated here https://github.com/elastic/kibana/pull/83507/commits/e3adf61761916c80021cd2e77d2bfbe077d16215